Add nailgun support to speed this up

diff --git a/confluence-cli-2.6.0/confluence.sh b/confluence-cli-2.6.0/confluence.sh
index 397d311..88182f5 100755
--- a/confluence-cli-2.6.0/confluence.sh
+++ b/confluence-cli-2.6.0/confluence.sh
@@ -4,4 +4,8 @@
 # - Customize for your installation, for instance you might want to add default parameters like the following:
 # java -jar `dirname $0`/lib/confluence-cli-2.6.0.jar --server http://my-server --user automation --password automation "$@"
 
-java -jar `dirname $0`/lib/confluence-cli-2.6.0.jar "$@"
+
+#java -jar `dirname $0`/lib/confluence-cli-2.6.0.jar "$@"
+#export CLASS_PATH=/usr/share/java/
+ng ng-cp `dirname $0`/lib/confluence-cli-2.6.0.jar
+ng org.swift.confluence.cli.ConfluenceClient "$@"
